src/views/collection-view/server.ts

```diff
@@ -26,7 +26,7 @@ export interface CollectionPageProps {
 
 export type CollectionLayout = Pick<
   SidebarSidecarLayoutProps,
-  'headings' | 'breadcrumbLinks'
```

Comment:
The `CollectionLayout` no longer uses `headings`, so I've dropped it from the type.